.github/workflows/test.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - main
+    tags:
```

Comment:
Shall we rename this file then? It’s not test anymore.

Reply:
yes, but i don't come up with a good name. Do you have any idea ?

Reply:
"Release testing", or just "test and release" lol.